src/main/java/com/google/graphgeckos/dashboard/datatypes/BuildInfo.java

```diff
@@ -62,9 +61,9 @@
    */
   public BuildInfo(GitHubData creationData) {
     this.commitHash = creationData.getCommitHash();
-    this.timestamp = Timestamp.of(new Date(creationData.getTimestamp()));
     this.branch = creationData.getBranch();
     this.builders = new ArrayList<>();
+    this.timestamp = Timestamp.parseTimestamp(creationData.getTimestamp());
```

Comment:
nit: try to keep initializations in the same order as the properties are listed inside the class, since it makes the code messy

Reply:
Done!